model/user/user_test.go

```diff
@@ -78,6 +79,75 @@ func (s *UserTestSuite) SetupTest() {
 	for _, user := range s.users {
 		s.NoError(user.Insert())
 	}
+
+	rm := GetRoleManager()
```

Comment:
is this mostly a gimlet test? Maybe the test could use the in-memory role manager?

Reply:
The test is for the implementation of the HasPermission function. It's true that the setup is entirely in gimlet interfaces but I think it still logically belongs here